schema/PARTNERSHIP_ATTRIBUTION_JF_TRANSFER_MEMO.json

```diff
@@ -414,7 +414,7 @@
             "type": "string",
             "minLength": 1,
             "maxLength": 100,
-            "pattern": "^JF Memo: [ -~]{0,65} \\(Partnership Attribution\\)$",
```

Comment:
All the following partnership jf transfer memos handle the truncation by truncating the contact rather than the whole description in the front end.  We don't need to change their validation because they already pass the stricter method

Reply:
Removed!